### ISSUE TITLE
typo in detailed descriptions of SRLI32.u

### ISSUE THREAD

**Issue by hope51607** [riscv/riscv-p-spec#9]
In section 6.39.2

```
Format:
SRLI32
...
SRAI32.u
...
```
should be
```
Format:
SRLI32
...
SRLI32.u
...
```

**Comment by chuanhua** [comment]
Thanks for finding this typo. I will fix it in the next release.

**Comment by chuanhua** [comment]
Fixed in the v0.9.1 release. (976dbe7)
